packages/arb-token-bridge-ui/src/util/deposits/helpers.ts

```diff
@@ -60,22 +69,11 @@ export const updateAdditionalDepositData = async (
 
   const { isClassic } = depositTx // isClassic is known before-hand from subgraphs
 
-  const isRetryableDeposit =
```

Comment:
Instead of assuming it's a retryable based on some conditions, we're now actually checking what object we're dealing with (`isEthDepositMessage`).